lib/css/components/tables.less

```diff
@@ -1,313 +1,296 @@
         thead th {
-            background-color: var(--black-050);
-        }
-        // Add in stripes
-        tbody tr:nth-child(even) {
-            background-color: var(--black-025);
+            a { //  If an anchor is used, it should appear like a normal header
+                color: inherit;
+            }
+
+            &.is-sorted { //  Selected state
+                color: var(--black-900);
+            }
+
+            color: var(--fc-light);
+            cursor: pointer;
         }
     }
 
-    //  $$  SPACING
-    //  ------------------------------------------------------------------------
-    //  --  SMALL
-    &.s-table__sm {
-        th,
-        td {
-            padding: var(--su4);
-        }
+    // Sizes
+    &&__sm {
+        --_ta-td-p: var(--su4);
+        --_ta-th-p: var(--su4);
     }
-    //  --  LARGE
-    &.s-table__lg {
-        th,
-        td {
-            padding: var(--su12);
-        }
+
+    &&__lg {
+        --_ta-td-p: var(--su12);
+        --_ta-th-p: var(--su12);
     }
 
-    //  $$  BORDERS
-    //  ------------------------------------------------------------------------
-    //  --  No Borders
-    &.s-table__b0 {
-        th,
-        td,
-        tr:last-of-type th,
-        tr:last-of-type td {
-            border-color: transparent;
+    // CHILD ELEMENTS
+    & &--cell {
+        .generate-cell-widths-classes(@n, @i: 1) when (@i =< @n) {
+            &@{i} { // generates iterated cell classes
+                @ta-cell-w: @ta-columns * @i;
+                --_ta-td-w: @ta-cell-w;
+                --_ta-th-w: @ta-cell-w;
+            }
+            .generate-cell-widths-classes(@n, (@i + 1));
         }
+  
+        .generate-cell-widths-classes(12);
+    }
 
-        //  Clear Header Styles
-        thead th {
-            background-color: transparent;
-            text-transform: initial;
-            font-size: inherit;
-            letter-spacing: initial;
+    & &--totals {
+        --_ta-td-fs: var(--fs-subheading);
+        --_ta-td-pt: var(--su12);
+        --_ta-td-fw: bold;
+        --_ta-th-fs: var(--fs-subheading);
+        --_ta-th-pt: var(--su12);
+    }
+
+    tbody {
+        + tbody { // If two table bodies are next to each other, visually separate them
+            border-top: var(--_ta-tbody-tbody-bw) solid var(--_ta-tbody-tbody-bc);
         }
 
-        //  This makes the border transparent, so we need to use whitespace
-        //  to achieve the same effect a 2px gray border achieves.
-        tbody + tbody {
-            border-color: transparent;
-            border-width: var(--su-static12);
+        th {
+            font-weight: normal;
         }
-    }
 
-    //  --  Horizontal Only with Table Outline
-    &.s-table__bx {
         tr {
-            > *:not(:first-child) {
-                border-left-color: transparent;
-            }
-            > *:not(:last-child) {
-                border-right-color: transparent;
+            &:nth-child(2n) {
```

Comment:
It looks like this selector has higher specificity (`.s-table tbody tr:nth-child(2n)`) now than the `.s-table tr.is-disabled` selector. This prevents the disabled `background-color` from being applied correctly for even rows.

<details>
    <summary>Before</summary>
    
<img width="774" alt="Screenshot 2022-12-13 at 13 38 21" src="https://user-images.githubusercontent.com/6231616/207320283-67a776b5-997c-4724-a673-0e29ed7ca151.png">
</details>

<details>
    <summary>After</summary>
    
<img width="774" alt="Screenshot 2022-12-13 at 13 39 38" src="https://user-images.githubusercontent.com/6231616/207320535-810d82c5-682c-4d6d-b16c-b2b777c53e89.png">
</details>

Reply:
Ah, yep! We need `.is-disabled` to also redefine the custom property setting the even row background color. On it!